lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -140,7 +140,7 @@
       <!-- <display file="genetrack.xml" /> -->
     </datatype>
     <datatype extension="gff" type="galaxy.datatypes.interval:Gff" display_in_upload="true" description="GFF lines have nine required fields that must be tab-separated." description_url="https://wiki.galaxyproject.org/Learn/Datatypes#GFF">
-      <converter file="gff_to_bed_converter.xml" target_datatype="bed"/>
+      <converter file="gff_to_bed_converter.xml" target_datatype="bed6"/>
```

Comment:
Would it be an option to add a gff -> bed12 converter?

Reply:
That would be the updated `tools/filters/gff2bed.xml` , but would only work if we move/copy/symlink it in `lib/galaxy/datatypes/converters/`
Ideally we could move it, but that would make the tool disappear from the panel (https://github.com/galaxyproject/galaxy/issues/9649)  and may break existing workflows (I haven't actually checked the latter).